lib/u3d/installation.rb

```diff
@@ -44,6 +44,39 @@ def self.create(root_path: nil, path: nil)
         WindowsInstallation.new(root_path: root_path, path: path)
       end
     end
+
+    def self.package_aliases
+      {
+        'Android' => [],
+        'iOS' => ['iPhone'],
+        'AppleTV' => ['tvOS'],
+        'Linux' => ['StandaloneLinux'],
+        'Mac' => %w[StandaloneOSXIntel StandaloneOSXIntel64 StandaloneOSX],
+        'Windows' => ['StandaloneWindows'],
+        'Metro' => [],
+        'UWP-IL2CPP' => [],
+        'Samsung-TV' => [],
+        'Tizen' => [],
+        'WebGL' => [],
+        'Facebook-Games' => ['Facebook'],
+        'Vuforia-AR' => ['UnityExtensions']
+      }
+    end
+
+    def packages
+      false
+    end
+
+    def package_installed?(package)
+      return false unless packages
+
+      return true if packages.include?(package)
+
+      aliases = Installation.package_aliases[package]
+      return false if aliases[package].nil?
```

Comment:
I don't think `aliases[package]` can get nil. You meant empty right?

In which case, we may just skip the line entirely as it is covered by the next one?

Reply:
I believe it can when the package is not known ie `aliases['NewPackage'] = nil`. It makes it so that we don't crash when some package is not known, which would make u3d unusable for versions with unknown packages. So this lane is mandatory imo.